frontend/src/components/organizations/NotForProfitQuestions.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <v-card class="my-4 pa-4 w-100" variant="outlined">
+    <v-container fluid class="pa-0">
+      <AppLabel>Not for Profit Information</AppLabel>
+      <v-card variant="outlined" class="soft-outline px-2 py-6 my-4">
+        <AppLabel>Date of Incorporation</AppLabel>
+
+        <AppDateInput id="date-to" v-model="model.dateOfIncorporation" :rules="[...rules.required, rules.MMDDYYYY]" :max="todayDate" :disabled="false" label="Select The Date" class="mt-6 mx-6" />
+      </v-card>
+      <v-card v-if="isReferenceLetterRequired" variant="outlined" class="soft-outline px-2 py-6 my-4 w-100">
+        <v-row no-gutters>
+          <AppLabel class="mr-4">Upload Documents</AppLabel>
+
+          <v-tooltip
+            content-class="tooltip"
+            text="A letter from a community representative or agency that has knowledge about the sector and can demonstrate how the society provides services to support the local community."
+            max-width="300px">
+            <template #activator="{ props }">
+              <v-icon size="large" v-bind="props">mdi-information-slab-circle-outline</v-icon>
+            </template>
+          </v-tooltip>
+        </v-row>
+        <v-card variant="outlined" class="soft-outline px-2 py-6 my-4 w-100">
+          <AppDocumentUpload
+            v-model="model.documentsToUpload"
+            :uploaded-documents="model.uploadedDocuments"
+            :document-type="DOCUMENT_TYPES.COMMUNITY_LETTER"
+            :document-label="DOCUMENT_LABELS.COMMUNITY_LETTER"
+            entity-name="accounts"
+            :disabled="false"
```

Comment:
disabled = false is redundant because it is default to false in the AppDocumentUpload

Reply:
sorry you totally said that before!!